### DISCUSSION TITLE
Remove dead space between song select carousel cards

### DISCUSSION THREAD

**Discussion by rszyma** [ppy/osu#28599] [Ideas] [answered]
Problem: In song select, there's a small gap between carousel cards that clicking on this small area does nothing. It's quite annoying as I land on it suprisingly too often, and it just doesn't feel good. 

https://github.com/ppy/osu/assets/44318430/30d5a992-908f-4906-8cb4-a03c616efa1e

Proposed solution: make the clicks on the gap result in clicking on nearest carousel item. In stable osu! there's no such gap, so this would be another step forward in the "matching stable feel" series.

related: https://github.com/ppy/osu/discussions/26198

**Comment by peppy** [comment, marked as answer]
I've attempted to improve things over at https://github.com/ppy/osu/pull/28601
